### ISSUE TITLE
Google Oauth | Access blocked: request is invalid error 400: invalid_request

### ISSUE THREAD

**Issue by supervision** [tokland/youtube-upload#375]
The upload was working pretty nifty for the last 3 years, 
but now I am running into a Google Oauth error:
Access blocked: request is invalid error 400: invalid_request

Did not changed anything in the shell script, which batch uploads the videos from the MacOS terminal.

I created new credentials and changed the old client_secrets.json,
but the error stays just the same.

Any help, gratefully received.

![Bildschirmfoto 2024-01-25 um 21 11 58](https://github.com/tokland/youtube-upload/assets/281370/ac9d57ae-771e-4d60-9439-543f25fccb31)

**Comment by jdanosborne** [comment]
Yes, getting the same thing. Seems to be down to Google tightening up on Oauth2 and disallowing OOB authentication - follow the links in the Access blocked dialog for more info.

I'm going to switch to https://developers.google.com/youtube/v3/guides/uploading_a_video

Dan
